### ISSUE TITLE
courses initial loading warning

### ISSUE THREAD

**Issue by RheuX** [open-learning-exchange/planet#7923]
![image](https://github.com/user-attachments/assets/3a924c82-d1d8-4845-9c7d-50f9f0b4fa36)

I'm wondering if anyone have this issue, 

<img width="533" alt="Screenshot 2024-12-11 at 10 16 14 AM" src="https://github.com/user-attachments/assets/eeeccc7f-4fdd-40dd-87bb-75916357770f" />

I'm getting a lot of this warning upon loading the courses which is making the courses loading screen stuck for a bit.

**Comment by Gavinp14** [comment]
I was not able to replicate this. Is it happening for you when you just try to view the courses?

**Comment by jessewashburn** [comment]
Yes, I am getting this too. When I view a course and then back out of it.
